examples/dnspolicy/script.sh

```diff
@@ -0,0 +1,5 @@
+kubectl create ns gateway
+envsubst < examples/dnspolicy/managedzone.yaml | kubectl apply -f -
+envsubst < examples/dnspolicy/gateway.yaml | kubectl apply -f -
+envsubst < examples/dnspolicy/dnspolicy.yaml | kubectl apply -f -
+envsubst < examples/dnspolicy/application.yaml | kubectl apply -f -
```

Comment:
All these example files seem specific to your setup e.g. `pb.hcpapps.net`. Should they be getting checked in?

Reply:
no, I'll remove them